docs/index.rst

```diff
@@ -39,10 +39,10 @@ which is explained in the `Developer Network Authentication Guide
 The SDK manages your API credentials for you. There are multiple ways to supply the SDK
 with your API credentials, which is explained in :ref:`authentication`.
 
-User Guide
-----------
+Get Started
```

Comment:
This reads a little strange maybe `Getting Started`?

Reply:
Yeah that works for me